tests/pretty_format_json_test.py

```diff
@@ -108,10 +108,8 @@ def test_badfile_main():
 
 
 def test_diffing_output(capsys):
-    resource_path = get_resource_path('not_pretty_formatted_json.json')
     expected_retval = 1
-    expected_diff = '''
-  {
+    expected_out = '''  {
```

Comment:
this seems weird to me that this is indented by two spaces instead of one -- just want to double check this

I'd also write this like:
```python
'''\
  {
...
'''
```

just so it's more clear what the actual string renders as

Reply:
Yeah I thought that was weird too, seems to be what difflib prints though so I went with it

Reply:
ahhhhh! it should be `difflib.unified_diff` I think!  my bad for suggesting `ndiff` 👍

Reply:
No worries! Problem is unified_diff outputs two lines that have trailing whitespace, which flake8 doesn't like. In my latest commit, I was forced to make the expected output string a little less readable in order for flake8 to pass the pre commit check - lmk what you think about that.